release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-de.html

```diff
@@ -27,7 +27,6 @@ <h2><b>Neue Funktionen</b></h2>
     <li>Linux - Synchronisierung von Links (Symlinks)</li>
     <li>Schutz hinzugef&uuml;gt, um zu verhindern, dass 2 Synchronisierungen innerhalb desselben Ordners stattfinden</li>
     <li>Die Protokolle werden jetzt 72 Stunden statt 24 Stunden aufbewahrt.</li>
```

Comment:
Regarding the client application logs, the purge frequency is 7 days and not 72 hours.
Regarding the server application logs, I don't know. @ChristopheLarchier @ClementKunz Is it parametrized in some configuration files?

Reply:
I updated the log purge frequency from 72 hours to 7 days in release notes (see commit: https://github.com/Infomaniak/desktop-kDrive/pull/84/commits/da5f0fa6fcabd6f56c0d10f1b8d2078319136458).

Reply:
IMHO both logs should depend on the same parameter